test/public/runs/overview.test.js

```diff
@@ -377,15 +377,28 @@ module.exports = () => {
                 const rowId = row.id;
                 return document.querySelector(`#${rowId}-definition-text`).innerText.split('\n')[0];
             }));
-            expect(definitions.length).to.equal(size);
```

Comment:
By removing this check. you might have runs that do not have the proper definition on other pages.

Reply:
no, this check checks nothing, because if `await waitForTableLength(page, size)` at line 374 passes, `definitions` list must have length equal `size`